symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/exception/InvalidJwtException.java

```diff
@@ -0,0 +1,15 @@
+package com.symphony.bdk.app.spring.exception;
+
+/**
+ * Thrown when the Jwt is not valid.
+ */
+public class InvalidJwtException extends RuntimeException {
```

Comment:
You could provide a "more" generic exception API, something like: 
```java
public class BdkAppException extends RuntimeException {
   private final BdkErrorCodeEnum error;
}
```
That would also simplify the controller advice